resources_pages/install_ds_stack_windows.md

```diff
@@ -538,39 +547,34 @@ named COPYING and the pdfTeX source.
 Primary author of pdfTeX: Han The Thanh (pdfTeX) et al.
 Compiled with libpng 1.6.37; using libpng 1.6.37
 Compiled with zlib 1.2.11; using zlib 1.2.11
-Compiled with xpdf version 4.02
-```
-
-The above is all we need to have LaTeX work with R Markdown documents, however for Jupyter we need to add several more packages. Before we do this, please [sign out of Windows](https://wiki.washjeff.edu/download/attachments/84967772/windows_10_logoff.png?version=1&modificationDate=1474137276320&api=v2) and back in again in order for the Git Bash terminal to be able to find the location of TinyTex.
-
-When you sign back in,
-install the additional LaTeX packages needed for Jupyter by pasting the following into the new terminal instance and press enter:
-by pasting the following:
-
-```
-tlmgr.bat install eurosym \
-  adjustbox \
-  caption \
-  collectbox \
-  enumitem \
-  environ \
-  fp \
-  jknapltx \
-  ms \
-  parskip \
-  pgf \
-  rsfs \
-  tcolorbox \
-  titling \
-  trimspaces \
-  ucs \
-  ulem \
-  upquote
-```
-
-To test that your latex installation is working with jupyter notebooks,
-launch `jupyter lab` from a terminal and open either a new notebook
-or the same one you used to test IRkernel above.
+Compiled with xpdf version 4.03
+```
+
+The above is all we need to have LaTeX work with R Markdown documents. However, for Jupyter we need to add several more packages. To complete the installation open an R console in JupyterLab, type the commands listed below and press `Ctrl` + `enter`:
+
+```
+tinytex::tlmgr_install(c("eurosym", 
```

Comment:
Why did you change this to run in the R console instead of a terminal? Does it have any advantage? I vaguely remember issues with not seeing the path correctly unless this was run from terminal.